wicamo96/FieldScout
# Login and registration

# Description

Foundational login. registration, and navbar functionality added.

## Type of change


# Testing Instructions

### Create database and populate data if necessary
- If you haven't created the FieldScout database, please do so in visual studio community.
- Use this to fill out tables and columns: [SQL data](https://github.com/wicamo96/FieldScout/blob/main/SQL/01_FieldScout_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/wicamo96/FieldScout/blob/main/SQL/02_FieldScout_Seed_Data.sql)

### Testing Log In and Register
- Click the execute with debugger solid green button to launch the backend in visual studio community.
- From the client directory in a git terminal, run `npm run dev`
- Once the Vite + React window pops up, you should be directed to the log in page
- Navigate to the register page via navbar or link at the bottom of the log in page
- Enter test data for each field. Once you click the register button you should be directed to the log in page
- Log in with your test credentials
- You will be taken to a blank dashboard page

# Checklist:
